### PR TITLE
pre1 stuff

### PR DESCRIPTION
**Pull request by Shnupbups**
idk

### CODE REVIEW

File: mappings/net/minecraft/client/options/GraphicsOption.mapping [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5365 net/minecraft/client/options/GraphicsOption
```
Comment (i509VCB) [inline mappings/net/minecraft/client/options/GraphicsOption.mapping]: I think another PR beat you to this

Comment (haykam821) [reply mappings/net/minecraft/client/options/GraphicsOption.mapping]: Conflicts with #1418

Comment (liach) [reply mappings/net/minecraft/client/options/GraphicsOption.mapping]: Yeah I personally favor #1418 and #1419 over this. This one conflicts with both

Comment (Shnupbups) [reply mappings/net/minecraft/client/options/GraphicsOption.mapping]: aight if that gets merged i'll remove from this

**Review by liach — APPROVED** [review]
Looks good, though might need to update later as this risks merge conflicts

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**